spec/requests/dataset_items_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+require 'rspec/mocks'
+
+describe "Dataset Items" do
+
+  create_entire_hierarchy
+
+  let(:dataset_item_attributes) {
+    FactoryGirl.attributes_for(:dataset_item, {audio_recording_id: audio_recording.id})
+  }
+
+  let(:update_dataset_item_attributes) {
+    {end_time_seconds: (dataset_item.end_time_seconds + 5) }
+  }
+
+  before(:each) do
+    @env ||= {}
+    @env['HTTP_AUTHORIZATION'] = admin_token
+
+    @create_dataset_item_url = "/datasets/" + dataset.id.to_s + "/items"
+    @update_dataset_item_url = "/datasets/" + dataset_item.dataset_id.to_s + "/items/" + dataset_item.id.to_s
```

Comment:
Can we use `url_helpers` here? Or any rails method for generating the route?
If not, if you use string interpolation (which should do the `.to_s` part for you).